### PR TITLE
Add key step

### LINKED ISSUE #2332: APIMatic integration docs not updated
While creating an APIMatic integration, there is a field to enter the APIMatic API key:
image

However, [the docs](https://learning.postman.com/docs/integrations/apimatic/) do not mention them.
As a comparison, the [Github integration docs](https://learning.postman.com/docs/integrations/github/#generating-a-github-personal-access-token) provide steps to generate the access token.

### PR DESCRIPTION
**Pull request by SueSmith**
Fixes #2332 

@claireriley gave this a very light redraft as well as adding the key info (just linked to apimatic docs)

### CODE REVIEW

File: src/pages/docs/integrations/apimatic.md [hunk 1]
```diff
@@ -29,35 +29,36 @@ If you don't already have a [GitHub account](https://github.com/), you'll need t
 
 ## Configuring APIMatic Integration
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select APIMatic from the list of third party integrations.
+In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select APIMatic from the list of third party integrations.
```
Comment (claireriley) [inline src/pages/docs/integrations/apimatic.md:32]: I might specify that the list of third party integrations is under **Browse Integrations** as I think the flow's a bit awkward.

File: src/pages/docs/integrations/apimatic.md [hunk 2]
```diff
@@ -29,35 +29,36 @@ If you don't already have a [GitHub account](https://github.com/), you'll need t
...
 
-    [![select apimatic](https://assets.postman.com/postman-docs/integrations_APImatic.png)](https://assets.postman.com/postman-docs/integrations_APImatic.png)
+<img alt="select apimatic" src="https://assets.postman.com/postman-docs/integrations_APImatic.png" width="200px"/>
 
-2. Click **View Details** to see information about APIMatic and how the Postman to APIMatic integration converts, formats, and saves your Postman collections to Github.
+Click **View Details** for more information on how the Postman to APIMatic integration converts, formats, and saves your collections to GitHub.
 
-   You can also click **Configured Integrations** to set up other integrations, view available integrations for APIMatic, or view all integrations.
+> You can alrernatively click **Configured Integrations** to access any existing integrations.
```
Comment (claireriley) [inline src/pages/docs/integrations/apimatic.md:38]: "alrernatively" lol

File: src/pages/docs/integrations/apimatic.md [hunk 3]
```diff
@@ -29,35 +29,36 @@ If you don't already have a [GitHub account](https://github.com/), you'll need t
...
 
-    [![configured integrations](https://assets.postman.com/postman-docs/WS-integrations-apimatic-conf-integr.png)](https://assets.postman.com/postman-docs/WS-integrations-apimatic-conf-integr.png)
+[![APIMatic Integration](https://assets.postman.com/postman-docs/apimatic-integration-overview.jpg)](https://assets.postman.com/postman-docs/apimatic-integration-overview.jpg)
```
Comment (claireriley) [inline src/pages/docs/integrations/apimatic.md:40]: I'd either make this image of the configured integrations tab or move it up to line 37

File: src/pages/docs/integrations/apimatic.md [hunk 4]
```diff
@@ -29,35 +29,36 @@ If you don't already have a [GitHub account](https://github.com/), you'll need t
...
+* Specify a directory in the repository where you want to add the collection.
+* Enter a filename for the directory in the repository.
+* Select the format in which you want to save the collection, such as Postman 2.0 or APIMatic format.
+* Specify the branch where you want to add the collection. _Note that the branch you want to back up your collection to must already exist._
 
-6. Click the **Add Integration** button.
+Click **Add Integration**.
 
-   APIMatic converts the collection’s documentation to your format of choice and pushes to your GitHub repository.  Now your Postman collection automatically saves to your GitHub repo every day, in whatever API description format you selected.  API format compatibility is moments away.
+APIMatic converts the collection documentation to your format of choice and pushes to your GitHub repository. Your Postman collection will automatically save to your GitHub repo every day, in the API description format you selected.
```
Comment (claireriley) [inline src/pages/docs/integrations/apimatic.md:64]: Small wording suggestions: "converts your collection documentation" and "pushes it to your GitHub repository"

**Comment by SueSmith** [comment]
Thanks @claireriley think I've addressed everything.. 🤦‍♀️

**Pull request merged**